roles/download/tasks/download_prep.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Register docker images info
+  raw: >-
+    {{ docker_bin_dir }}/docker images -q | xargs {{ docker_bin_dir }}/docker inspect -f "{{ '{{' }} (index .RepoTags 0) {{ '}}' }},{{ '{{' }} (index .RepoDigests 0) {{ '}}' }}" | tr '\n' ','
+  no_log: true
+  register: docker_images
+  failed_when: false
+  changed_when: false
+  check_mode: no
+
+- name: container_download | Create dest directory for saved/loaded container images
+  file:
+    path: "{{local_release_dir}}/containers"
+    state: directory
+    recurse: yes
+    mode: 0755
+    owner: "{{ansible_ssh_user|default(ansible_user_id)}}"
+
+# This is required for the download_localhost delegate to work smooth with Container Linux by CoreOS cluster nodes
+- name: container_download | Hack python binary path for localhost
+  raw: sh -c "mkdir -p /opt/bin; ln -sf /usr/bin/python /opt/bin/python"
+  delegate_to: localhost
+  when: download_delegate == 'localhost'
```

Comment:
do we need to add an `and os == containerlinux`?

Reply:
Starting in ansible 2.3 we can use `ansible_playbook_python` instead of hacking this.